agent/util-scripts/pbench-register-tool

```diff
@@ -202,7 +205,11 @@ if [[ -z ${name} ]]; then
 	exit 1
 fi
 if [[ ! -x "${pbench_bin}/tool-scripts/${name}" ]]; then
```

Comment:
I suppose this will catch the case where `name` is an error message, but it won't look pretty.